miso-web/src/main/webapp/scripts/hot_sample.js

```diff
@@ -312,22 +312,23 @@ HotTarget.sample = (function() {
               } // else externalName will come from an existing Identity via the Identity Alias column
             },
             depends: 'identityAlias',
-            update:function(sam, flat, flatProperty, value, setReadOnly, setOptions, setData) {
-              if(!value || value == "Delete external name, select a project, then re-enter external name." || value == '(...searching...)'
-                || /First Receipt \(.*\)/.test(value) || !flat['externalName']){
+            update: function(sam, flat, flatProperty, value, setReadOnly, setOptions, setData) {
+              if (!value || value == "Delete external name, select a project, then re-enter external name." || value == '(...searching...)'
+                  || /First Receipt \(.*\)/.test(value) || !flat['externalName']) {
                 setOptions({
                   'renderer': Handsontable.renderers.TextRenderer
                 });
                 return;
-              };
-              
+              }
+              ;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1116)